From 81a1f4a5ffab6a20f5fa90c722791e6956999988 Mon Sep 17 00:00:00 2001
Message-Id: <81a1f4a5ffab6a20f5fa90c722791e6956999988.1389014116.git.minovotn@redhat.com>
In-Reply-To: <c8cc35838d42aa286242772d97e3a9be7bb786ba.1389014116.git.minovotn@redhat.com>
References: <c8cc35838d42aa286242772d97e3a9be7bb786ba.1389014116.git.minovotn@redhat.com>
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Mon, 9 Dec 2013 14:09:20 +0100
Subject: [PATCH 32/50] block/iscsi: remove .bdrv_has_zero_init

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1386598178-11845-35-git-send-email-pbonzini@redhat.com>
Patchwork-id: 56071
O-Subject: [RHEL 7.0 qemu-kvm PATCH 34/52] block/iscsi: remove .bdrv_has_zero_init
Bugzilla: 1007815
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

From: Peter Lieven <pl@kamp.de>

since commit 3ac21627 the default value changed to 0.

Signed-off-by: Peter Lieven <pl@kamp.de>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 4b52498e62d02a30f780875f54431d1bad0bcabd)
---
 block/iscsi.c | 7 -------
 1 file changed, 7 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block/iscsi.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/block/iscsi.c b/block/iscsi.c
index 3e3ae64..c5cd86d 100644
--- a/block/iscsi.c
+++ b/block/iscsi.c
@@ -1507,11 +1507,6 @@ static int iscsi_truncate(BlockDriverState *bs, int64_t offset)
     return 0;
 }
 
-static int iscsi_has_zero_init(BlockDriverState *bs)
-{
-    return 0;
-}
-
 static int iscsi_create(const char *filename, QEMUOptionParameter *options,
                         Error **errp)
 {
@@ -1607,8 +1602,6 @@ static BlockDriver bdrv_iscsi = {
     .bdrv_aio_writev = iscsi_aio_writev,
     .bdrv_aio_flush  = iscsi_aio_flush,
 
-    .bdrv_has_zero_init = iscsi_has_zero_init,
-
 #ifdef __linux__
     .bdrv_ioctl       = iscsi_ioctl,
     .bdrv_aio_ioctl   = iscsi_aio_ioctl,
-- 
1.7.11.7

